third_party/mesa/src/util/simple_mtx.h

```diff
@@ -28,6 +28,10 @@
 #include "util/macros.h"
 #include "util/u_call_once.h"
 #include "u_atomic.h"
+// GOOGLE: Include windows header when the OS is Windows.
+#ifdef _WIN32
```

Comment:
out of curiosity, is there a reason you use _WIN32 here instead of _MSC_VER?

Reply:
It should've been _MSC_VER to be more exact but works in a similar way. Removed since it is no longer needed.